### PR TITLE
fix: add <Ajay84sia>

### LINKED ISSUE #1691: [PROFILE ADDITION] <Ajay>
### Profile screenshot

profile-readme

### PR DESCRIPTION
**Pull request by Ajay84sia**
resolve #1691

### CODE REVIEW

File: profiles/Ajay84sia.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/98752820/215834058-27ffacf5-9525-43ce-8ca2-771551b2969c.png",
+  "issueId": 70,
```
Comment (Panquesito7) [inline profiles/Ajay84sia.json:3]: Wrong issue ID.
```suggestion
  "issueId": 1691,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Ajay84sia** [comment]
@Panquesito7  my  "issueId": 1691,

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**